libs/ui/src/tokens/components/_form-control.css

```
@@ -53,6 +54,13 @@
     2 +
     1lh
   );
+  --height-form-control-lg: calc(
+    var(--border-width-form-control) *
```

Stylelint (17.6.0)

[error] Unexpected newline after "*" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)